server/ladder_service.py

```diff
@@ -289,34 +287,35 @@ def inform_player(self, player: Player):
                     )
                 })
 
-    def start_queue_handlers(self):
-        for queue in self.queues.values():
-            asyncio.ensure_future(self.handle_queue_matches(queue))
+    def on_match_found(
+        self,
+        s1: Search,
+        s2: Search,
```

Comment:
for me, s1 and s2, etc is very bad naming. Don't name variables by type, name them by meaning

Reply:
There is no other meaning. It's search 1 and search 2. Just like player 1 and player 2 except it's not just one player, it's a search.